src/status_im/chat/models/delete_message_for_me.cljs

```diff
@@ -0,0 +1,69 @@
+(ns status-im.chat.models.delete-message-for-me
+  (:require [re-frame.core :as re-frame]
+            [status-im.ethereum.json-rpc :as json-rpc]
+            [status-im.utils.fx :as fx]
+            [taoensso.timbre :as log]))
+
+(defn- update-db-clear-delete-message-for-me-undo-timer
+  [db chat-id message-id]
+  (when (get-in db [:messages chat-id message-id])
+    (update-in db
+               [:messages chat-id message-id]
+               dissoc
+               :deleted-for-me-undoable-till)))
+
+(defn- update-db-delete-message-for-me-locally
```

Comment:
You could take advantage of Clojure's namespacing capabilities and not repeat the words `delete-message-for-me` in all vars, given the namespace already says it's all related to `delete-message-for-me`. For example (just a suggestion for the names):

- `update-db-delete-message-for-me-locally` becomes `update-db-locally`
- `update-db-clear-delete-message-for-me-undo-timer` becomes `update-db-clear-undo-timer`
- `undo-delete-message-for-me` becomes `undo`
- `delete-message-for-me` becomes `delete`
- `delete-message-for-me-and-sync` becomes `delete-and-sync`

Reply:
done